### PR TITLE
Change approach to pagination

### LINKED ISSUE #95: Offset doesn't reset when initiating a new query
On the getRecordings page, if you are looking at a table with offset>0 and then run a different query, it will retain the offset and may potentially not display any results.

Perhaps best behaviour is for offset to reset to zero is the count is less than the offset, and then run the query again (so that some results will always be displayed)

### PR DESCRIPTION
**Pull request by arthurmcgregor**
- Hold page state in #pageNumber
- Use limit and pageNumber to determine offset
- Replace inc() and dec() with changePage("+1") and changePage("-1")
- Add pageButtons(page) to update pageNumber and show/hide next/prev 
buttons
- Update sendQuery(page) to allow a specific page to be queried (and it 
will determine the offset automatically using limit)

Closes #95

### CODE REVIEW

File: public/javascripts/getRecordings.js [hunk 1]
```diff
@@ -441,28 +448,45 @@ function clearTable() {
...
+  // Show/hide page buttons
+  let limit = Number(document.getElementById('limit').value);
+  if (page === 1) {
+    // Hide previous page button when on page 1
+    changeVisibility(document.getElementsByClassName('prevPage'), "hidden");
+    changeVisibility(document.getElementsByClassName('pageLabel'), "visible");
+    changeVisibility(document.getElementsByClassName('nextPage'), "visible");
+  } else if (page * limit > count) {
+    // Hide next page button when on final page
+    changeVisibility(document.getElementsByClassName('nextPage'), "hidden");
```
Comment (CameronRP) [inline public/javascripts/getRecordings.js]: If there are only 2 pages of results the previous page button is not visible on the second page. Just adding  `changeVisibility(document.getElementsByClassName('prevPage'), "visible");` should fix this I think.

File: public/javascripts/getRecordings.js [hunk 2]
```diff
@@ -441,28 +448,45 @@ function clearTable() {
...
+  pageNumber.innerText = page;
+  let pageNumberBottom = document.getElementById('pageNumberBottom');
+  pageNumberBottom.innerText = page;
+  // Show/hide page buttons
+  let limit = Number(document.getElementById('limit').value);
+  if (page === 1) {
+    // Hide previous page button when on page 1
+    changeVisibility(document.getElementsByClassName('prevPage'), "hidden");
+    changeVisibility(document.getElementsByClassName('pageLabel'), "visible");
+    changeVisibility(document.getElementsByClassName('nextPage'), "visible");
```
Comment (CameronRP) [inline public/javascripts/getRecordings.js]: Will need to check if there is only one page of results as we don't want to show nextPage when there is only one page.

**Review by CameronRP — CHANGES_REQUESTED** [review]
Just a couple of things to fix.

**Comment by arthurmcgregor** [comment]
Thanks @CameronRP Hopefully fixed now

**Review by mjs — APPROVED** [review]
I've tried out these changes and this is much better.

**Review by CameronRP — APPROVED** [review]

**Pull request merged**